content/nginxaas-azure/faq.md

```diff
@@ -17,10 +17,10 @@ Common questions about F5 NGINX as a Service for Azure (NGINXaaS).
 - The usage data is reported to NGINXaaS. 
 
 ### Is NGINXaaS available in my subscription or in F5 subscription?
-- Your NGINXaaS deployment resource is visible to you under your subscription. The underlying compute resources of your deployment, which are managed by NGINX on your behalf, are not visible in your subscription.
+- Your NGINXaaS deployment resource is visible to you under your Azure subscription. The underlying resources of any NaaS deployment, which are managed by NGINX on your behalf, are not visible in your subscription.
```

Comment:
We have only referred to our service as `NGINXaaS` in these docs and have not used `NaaS` - perhaps we can maintain that convention here as well.